app/src/main/java/lt/vilnius/tvarkau/utils/NetworkUtils.java

```diff
@@ -0,0 +1,13 @@
+package lt.vilnius.tvarkau.utils;
+
+import android.app.Activity;
+import android.content.Context;
+import android.net.ConnectivityManager;
+
+public class NetworkUtils {
+
+    public static boolean isNetworkConnected(Activity activity) {
+        ConnectivityManager cm = (ConnectivityManager) activity.getSystemService(Context.CONNECTIVITY_SERVICE);
+        return cm.getActiveNetworkInfo() != null;
```

Comment:
I think this can be not robust solution as `getActiveNetworkInfo()` can return `null` in some condtions.   https://developer.android.com/reference/android/net/ConnectivityManager.html#getActiveNetworkInfo()

Also this method can return true if we for example connect to wireless network but it doesn't have internet access.